packages/frontend/src/views/CalendarView/CalendarView.module.css

```diff
@@ -0,0 +1,51 @@
+.sidebarHeadings {
+  font-size: var(--flocker-font-size-medium);
+  font-weight: var(--flocker-font-weight-light);
+  margin-top: 16px;
+}
+
+.timeMatcher {
+  width: 60vw;
+  height: 70vh;
+}
+
+h2 {
+  font-size: var(--flocker-font-size-body);
+  font-weight: var(--flocker-font-weight-light);
+  padding: 16px 0 8px 0;
+}
+
+h3 {
+  font-size: var(--flocker-font-size-small);
+  font-weight: var(--flocker-font-weight-light);
+  padding: 0 10px 0 10px;
+}
+
+.legendKeys {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.circleBothAvailable {
+  width: 2rem;
+  height: 2rem;
+  background-color: var(--flocker-color-tertiary);
+  opacity: 70%;
+  border-radius: 50%;
+}
+
+.circleUserAvailable {
+  width: 2rem;
+  height: 2rem;
+  background-color: var(--flocker-color-secondary);
+  opacity: 70%;
+  border-radius: 50%;
+}
+.circleFlockAvailable {
```

Comment:
Lots of duplicate styling. You could create a common class with styling, and then these classes can set a variable for the color for the common class to use e.g.

```css
.circleBothAvailable {
  --flocker-circle-availability-color: var(--flocker-color-tertiary);
}

...

.circleAvailability {
  width: 2rem;
  height: 2rem;
  background-color: var(--flocker-circle-availability-color);
  opacity: 70%;
  border-radius: 50%;
}
```

Reply:
fixed!